### ISSUE TITLE
Printability: Horizontal Tunnels

### ISSUE THREAD

**Issue by Schreibmaschine-Berlin** [gestalte-design/zine-machine#2]
## Issue
This print has many internal horizontal cylinders which are hard to print, even on well calibrated printers. There are two main reasons for this especially when relying on the horizontal tunnels to act as hinges as they are here.

1. The stair stepping inherent to FDM printing on any curve along the XZ and YZ planes. Ideally if you are printing hinges it works best to place any smooth curves along the XY planes. these stair steps will cause ratcheting and jamming on the hinge.
2. the 90° overhang at the top of the cylinder will always drooping

In the photo attached you can see the stair stepping, drooping, and a bulge along the thin edge as well. This print was made with a 0.8mm nozzle which has an even higher tollerance for overhang than your recommended 0.4 nozzle

![PXL_20210301_083338101](https://user-images.githubusercontent.com/4319326/109471722-74899980-7a71-11eb-8205-6bdb0f9a594e.jpg)

## Recommendations. 
For the hinges, the hinge blocks should be printed separately in a different orientation so the cylinder is vertical.

For the tunnels inside of the letters I recommend an alignment hole with a peak. This is a common solution that does not have more than a 45° overhang

![file-D5MyhkCNqj](https://user-images.githubusercontent.com/4319326/109473646-da772080-7a73-11eb-95e5-d181629d7df7.jpg)

**Comment by gavin-owens** [comment]
The less moving parts the better, in my opinion, so I'm hesitant to move the hinge blocks into a separate piece. Right? Consider we can get a better print, which may have a nicer overall movement and feel to it, but introduces more complexity in the assembly and potentially some weakness into a part of the machine that will be exercised regularly. I don't have any good alternative suggestions here, unfortunately, but overall I think you can get most of the way there with some finishing on the hinges.

As for the alignment hole with the peak, seems like a great suggestion to me. Do you want to test it out on a few letters and see how it works?

**Comment by Schreibmaschine-Berlin** [comment]
Yeah I don't mind going in with a drill bit and cleaning up tunnels but some parts (body.stl) have the tunnels positioned in a way which is inaccessible.

I haven't printed the lettersets yet since it is a 2 day print at the .25mm nozzle. (the 0.4 isn't able to print all the letters)

**Comment by Schreibmaschine-Berlin** [comment]
This is how I would redesign the hinge blocks. I respect the decision not to use external hardware like screws since they vary globally but I think glue is ok to call for but may not even be necessary if you can get a good friction fit. I have lots of designs that use friction fit butterfly joints and they work fine. I think the benefits of smoother operation and repairability if one of the hinge pins breaks far outweighs the cost of a few extra parts.
![PXL_20210305_110118632](https://user-images.githubusercontent.com/4319326/110108378-6955a700-7dac-11eb-81d2-c063a7fc7fe6.jpg)

It's also the case that you can still print the hinge pins horizontally since as long as the tunnel is printed vertically and smooth you will not get the grinding of layers. And since the hinge assembly can be built and then attached to the body you could have a longer single hinge pin and avoid all the fiddly pin assembly so the part count would go down actually.

**Comment by gavin-owens** [comment]
Hm, ok, I think you make some really good points. I generally prefer friction fits, because they are more immediate — plus we're trying to avoid the need to acquire any other material for assembly if possible (why we got rid of these tiny springs used in the assembly). 

Will bring this proposal to Constantine and see if he has any recommendations. Will try to get back to you in a couple days. Thanks for your patience!

**Comment by gavin-owens** [comment]
Hm, ok, I think you make some really good points. I generally prefer friction fits, because they are more immediate — plus we're trying to avoid the need to acquire any other material for assembly if possible (why we got rid of these tiny springs used in the assembly). 

Thanks for your patience!

So, Constantine claims he was able to print without any overhang on the hinges. Have you tried to print with the 0.4 nozzle?
